src/routes/index.js

```diff
@@ -1,8 +1,16 @@
-import express from 'express'
-import { getMathRoutes } from './math'
-
-export const getRoutes = () => {
```

Comment:
Il motivo per cui Kent aveva fatto questa funzione `getRoutes` è per motivi di riproducibilità, e testing, così come hai rifattorizzato il modulo, dovrebbe funzionare lo stesso, ma così il primo import a `routes/index.js` scatenerà immediatamente la creazione del router e dal quel momento in poi si avrà sempre la stessa reference del router (che potrebbe essere quello che vogliamo eh 🤔 )

Reply:
ok capito